### PR TITLE
Fix header spacing on profile tabs

### PR DESCRIPTION
**Pull request by conrad-vanl**
Fixes issue where type refactor squashed the header spacing.

## Creator

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [x] Review function and form on Web
- [x] Review new test logic

### CODE REVIEW

**Comment by burkeshartsis** [comment]
Looks like there is a bug where the profile pic isn't shown if you haven't liked something yet. 
### No likes
simulator screen shot - iphone x - 2018-02-05 at 18 31 51
### With Likes
simulator screen shot - iphone x - 2018-02-05 at 18 32 42

**Comment by burkeshartsis** [comment]
Android has two profile images or something
screenshot_1517873767

**Comment by conrad-vanl** [comment]
> Android has two profile images or something

This has to do with android not supporting `BlurView`, and will _not_ be fixed in this PR. once profile image uploading is supported I'll clean up this and other bugs that has to do with the profile image view. I will fix the other issue you found in this PR though (header not showing up unless you have likes)

**Comment by conrad-vanl** [comment]
Alright @burkeshartsis the bug you came across should be fixed, the review on this PR can be finished.

File: src/tabs/Profile/Likes.js [hunk 1]
```diff
@@ -10,14 +10,19 @@ import FeedView from '@ui/FeedView';
...
   textAlign: 'center',
+});
+
+const TextWrapper = styled(({ theme }) => ({
   paddingHorizontal: theme.sizing.baseUnit,
   paddingVertical: theme.sizing.baseUnit,
-}));
+}))(View);
 
 const RecentLikesHeaderText = asHeaderText(H7);
```
Comment (burkeshartsis) [inline src/tabs/Profile/Likes.js]: that's neat!

**Pull request merged**